c_src/membrane_g711_ffmpeg_plugin/decoder.spec.exs

```diff
@@ -0,0 +1,20 @@
+module Membrane.G711.FFmpeg.Decoder.Native
+
+state_type "State"
+
+spec create() :: {:ok :: label, state} | {:error :: label, reason :: atom}
+
+spec decode(payload, state) ::
+       {:ok :: label, frames :: [payload]}
+       | {:error :: label, reason :: atom}
+
+spec flush(state) ::
+       {:ok :: label, frames :: [payload]}
+       | {:error :: label, reason :: atom}
+
+spec get_metadata(state) ::
+       {:ok :: label, sample_fmt :: atom}
+       | {:error :: label, :sample_fmt :: label}
+
+# TODO: do we need this?
+# dirty :cpu, decode: 2, flush: 1
```

Comment:
https://hexdocs.pm/unifex/Unifex.Specs.DSL.html#dirty/2

No idea if we should keep it or not. `h264_ffmpeg_plugin` has these lines in both encoder and decoder spec, but the arities passed to the `dirty` macro don't match the true arities of functions...

Reply:
After consulting with other members of the team, we'll keep it